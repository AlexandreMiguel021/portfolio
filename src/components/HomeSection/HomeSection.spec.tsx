import { render } from '@testing-library/react'
import { HomeSection } from '.'
import '@testing-library/jest-dom'

describe('HomeSection', () => {
	it('should render a HomeSection correctly', () => {
		const { container } = render(<HomeSection />)
		expect(container).toMatchSnapshot()
	})
})
