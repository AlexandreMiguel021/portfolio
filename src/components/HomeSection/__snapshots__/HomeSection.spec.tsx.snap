// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeSection should render a HomeSection correctly 1`] = `
<div>
  <section
    class="section"
    id="home"
  >
    <div
      class="container"
    >
      <div
        class="wrapper"
        data-aos="fade-up"
      >
        <div
          class="introduction"
        >
          <h1
            class="name"
          >
            Olá, eu sou o 
            <span>
              Alexandre Miguel
            </span>
          </h1>
          <p
            class="role"
          >
            Desenvolvedor Front-end
          </p>
          <div
            class="photo"
            data-aos="fade-up-left"
          >
            <span
              style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
            >
              <span
                style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
              >
                <img
                  alt=""
                  aria-hidden="true"
                  src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%27500%27%20height=%27500%27/%3e"
                  style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
                />
              </span>
              <img
                alt="Um rapaz jovem de moletom e óculos de grau"
                data-aos="fade-zoom-in"
                data-nimg="intrinsic"
                decoding="async"
                src="/_next/image?url=%2Fimages%2Fme-bg-hidden.png&w=1080&q=75"
                srcset="/_next/image?url=%2Fimages%2Fme-bg-hidden.png&w=640&q=75 1x, /_next/image?url=%2Fimages%2Fme-bg-hidden.png&w=1080&q=75 2x"
                style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
              />
            </span>
          </div>
          <p>
            Apaixonado por tecnologia e inovação, estou sempre ansioso para ouvir sobre novas ideias, criar e aprender coisas novas.
          </p>
          <p
            class="info"
          >
            O código fonte deste site está disponível em meu 
            <a
              href="https://github.com/AlexandreMiguel021/portfolio"
            >
              Github
            </a>
          </p>
          <a
            class="component large"
            href="/document/CV-Alexandre-Front-end.pdf"
            target="_blank"
          >
            <span
              class="icon"
            >
              <svg
                fill="currentColor"
                height="1em"
                viewBox="0 0 16 16"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M4 8a.5.5 0 0 1 .5-.5h5.793L8.146 5.354a.5.5 0 1 1 .708-.708l3 3a.5.5 0 0 1 0 .708l-3 3a.5.5 0 0 1-.708-.708L10.293 8.5H4.5A.5.5 0 0 1 4 8z"
                  fill-rule="evenodd"
                />
              </svg>
            </span>
            <span
              class="text"
            >
              Ver meu currículo
            </span>
          </a>
        </div>
      </div>
    </div>
  </section>
</div>
`;
